src/handleAction.js

```diff
@@ -12,10 +14,18 @@ export default function handleAction(actionType, reducers, defaultState) {
     : [reducers.next, reducers.throw].map(reducer => (isNil(reducer) ? identity : reducer));
 
   return (state = defaultState, action) => {
+    if (!isValidFSA(action)) {
+      throw new Error('The FSA spec mandates an action object with a type. Try using the createAction(s) method.');
+    }
+
     if (!includes(actionTypes, action.type.toString())) {
       return state;
     }
 
     return (action.error === true ? throwReducer : nextReducer)(state, action);
   };
 }
+
+function isValidFSA(action) {
```

Comment:
Sounds good. Though, we may be dropping Symbol support soon https://github.com/acdlite/redux-actions/issues/126.